src/Distances.jl

```diff
@@ -16,6 +16,8 @@ export
     pairwise!,
     evaluate,
 
+    fastview,
```

Comment:
We probably don't want to export this

Reply:
I think we should either export this, or a different way of accessing the allocation-free distance evaluation of matrix columns, or possibly both.

This was my original problem after all: I need to compute distances between more-or-less random columns in an inner loop (think ball-trees).

Would you feel better if I patched in bounds-checking for the fastview? Would be a no-op in inbounds contexts anyway...

Also, users who define their own distance functions get exposed to the fastview anyway (in their evaluate-function, if they call colwise or pairwise).